src/main/scala/dsls/regex/RegularExpression.scala

```diff
@@ -19,6 +34,15 @@ object EMPTY extends RegularExpression
 /** a regular expression that matches the empty string */
 object EPSILON extends RegularExpression
 
+/** implicit conversions */
+object RegularExpression {
+  implicit def CharacterToLiteral (c : Char) = new Literal(c)
+  
+  implicit def StringToReg (s : String) : RegularExpression = 
+    if (s.isEmpty()) EPSILON
+    else new Concat(s.head, StringToReg(s.tail))
```

Comment:
You don't actually need to call `StringToReg` here; you've already defined the implicit conversion :smiley: